spec/features/finding/index_details_spec.rb

```diff
@@ -2,38 +2,61 @@
 
 RSpec.feature "Index details" do
   scenario do
-    given_there_is_an_edition
+    given_there_is_an_edition_in_my_organisation
+    and_an_an_edition_not_in_my_organisation
     when_i_visit_the_index_page
-    then_i_can_see_the_title
-    and_i_can_see_the_document_type
-    and_i_can_see_the_state
-    and_i_can_see_who_last_edited_it
+    then_i_am_redirected_to_my_organisation_filter
+    and_i_can_see_the_edition_title
```

Comment:
Just a minor thing, but since we're changing these steps, could we just combine into `i_can_see_the_edition`?

Reply:
yeah suppose so, will mean a few assertions.